lib/gtfs_realtime_viz.ex

```diff
@@ -94,6 +157,17 @@ defmodule GTFSRealtimeViz do
     end)
   end
 
+  @spec format_times([String.t] | nil) :: [String.t]
+  def format_times(nil) do
+    []
+  end
+  def format_times(time_list) do
+    time_list
+    |> Enum.sort()
+    |> Enum.take(2)
+    |> Enum.map(& "#{&1} seconds")
```

Comment:
I think @dmaltzan had talked about wanting the datetimes rather than the seconds?

Reply:
I had mentioned that, but actually seconds is OK as long as it's measured from the same baseline value of "now" for each